EnglishFilter/sections/specific.txt

```diff
@@ -256,7 +256,12 @@ feetporno.com,familyporn.tv,freehardcore.com,webvoyeur.com,moviesand.com,momvids
 !
 !
 !
-uwatchfree.*##a[href*="https://www.topcpmnetwork.com/"]
+uwatchfree.tech##a[href*="https://www.topcpmnetwork.com/"]
+||hiphopza.com/wp-content/uploads/*/ad.gif
+||wp.com/hiphopza.com/wp-content/uploads/*/ad.gif$domain=hiphopza.com
+||cdjapan.co.jp/aff/tmpl/big_banner_728x90i.js
+thefmovies.me##a[href^="http://viewlnk.com/"][target="_blank"]
+codesnail.com##.inside-right-sidebar > #custom_html-2
```

Comment:
To avoid coflicts like that you shouldn't add rules at the beginning, just a little lower

Reply:
@zloyden What conflict?